dht_filters.go

```diff
@@ -217,7 +217,7 @@ func isEUI(ip net.IP) bool {
 
 func sameV6Net(a, b net.IP) bool {
 	//lint:ignore SA1021 We're comparing only parts of the IP address here.
-	return len(a) == net.IPv6len && len(b) == net.IPv6len && bytes.Equal(a[0:8], b[0:8]) //nolint
+	return len(a) == net.IPv6len && len(b) == net.IPv6len && bytes.Equal(a[0:8], b[0:8]) // nolint
```

Comment:
https://github.com/golangci/golangci-lint/blob/d286cb910c4fb9f91d26e630e033b05ae3bfd9db/docs/src/docs/usage/false-positives.mdx#nolint-directive

> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

TIL, thanks!